packages/babel-generator/src/node/parentheses.js

```diff
@@ -50,7 +50,7 @@ export function ObjectExpression(node: Object, parent: Object, printStack: Array
     return true;
   }
 
-  return isFirstInStatement(printStack);
+  return isFirstInStatement(printStack, true);
```

Comment:
Does this have to go in `isFirstInStatement` or can it be it's own if check above this line?